.vscode/launch.json

```diff
@@ -13,7 +13,7 @@
 				"transforms/__tests__/codemod-missing-await-act.js"
 			],
 			"console": "integratedTerminal",
-			"internalConsoleOptions": "neverOpen"
+			"runtimeExecutable": "${userHome}/.volta/bin/node"
```

Comment:
Needed to run on my MBP2019. This looks like it should work on Linux as well but both require a hardcoded VOLTA_HOME to `~/.volta`